src/network/if_nameindex.c

```diff
@@ -54,7 +54,7 @@ static int netlink_msg_to_nameindex(void *pctx, struct nlmsghdr *h)
 			map = &ctx->list[i-1];
 			if (map->index == index &&
 			    map->namelen == namelen &&
-			    memcmp(map->name, RTA_DATA(rta), namelen) == 0)
+			    memcmp((void *)map->name, RTA_DATA(rta), namelen) == 0)
```

Comment:
`mem*` functions do not accept checked pointers yet.

Reply:
`mem*` functions should now accept checked pointers. See https://github.com/microsoft/checkedc-musl/commit/7d1fec8b8f179b715370c26f17268898ad04cf5a

Reply:
Thanks! I removed the cast.